config/kernelci.toml

```diff
@@ -14,7 +14,7 @@ output = "/home/kernelci/data/output"
 storage_config = "docker-host"
 
 [scheduler]
-output = "/home/kernelci/output"
+output = "/home/kernelci/data/output"
```

Comment:
Is this working as expected?
I guess we also need to adjust the mounted volume `- './data/output:/home/kernelci/output'` for scheduler service to make this work.

Reply:
Good point, i will prepare fresh VM to retest all this changes from zero